libkbfs/quota_usage.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"golang.org/x/net/context"
+
```

Comment:
Most of the time we don't have separate import blocks here, but I don't feel too strongly about it.